src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -96,6 +97,16 @@ public Conversation getConversationWithTitle(String title) {
     return null;
   }
 
+  /** Find and return the Conversation with the given UUID. */
+  public Conversation getConversationWithId(UUID id) {
```

Comment:
Please add a unit test for this method

Reply:
Unit test has now been added for this method.